src/constants/formItemTexts.ts

```diff
@@ -0,0 +1,13 @@
+const FORM_ITEM_TEXTS = {
+  formTitle: "새로운 음식점",
+  categoryTitle: "카테고리",
+  nameTitle: "이름",
+  distanceTitle: "거리(도보 이동 시간)",
+  descriptionTitle: "설명",
+  descriptionDescription: "메뉴 등 추가 정보를 입력해 주세요",
+  urlTitle: "참고 링크",
+  urlDescription: "매장 정보를 확인할 수있는 링크를 입력해 주세요.",
+  cancelButton: "취소하기",
+  addButton: "추가하기",
+};
```

Comment:
아래처럼 작성하면 지금 혜인이 작성한 것과 어떤 차이가 있을까요? 한 번 찾아보셔도 좋을거 같습니다 ㅎㅎ

```suggestion
const FORM_ITEM_TEXTS = {
  formTitle: "새로운 음식점",
  categoryTitle: "카테고리",
  nameTitle: "이름",
  distanceTitle: "거리(도보 이동 시간)",
  descriptionTitle: "설명",
  descriptionDescription: "메뉴 등 추가 정보를 입력해 주세요",
  urlTitle: "참고 링크",
  urlDescription: "매장 정보를 확인할 수있는 링크를 입력해 주세요.",
  cancelButton: "취소하기",
  addButton: "추가하기",
} as const
```

Reply:
`Const Assertions`
: as const를 사용하면 TypeScript는 객체의 모든 필드를 readonly 로 만들고, 배열을 readonly로 만들며, 리터럴 값은 그 값 자체의 타입으로 추론한다. 

as const를 해주지 않으면 타입을 `string`으로 추론하지만
<img width="433" alt="스크린샷 2024-03-09 오후 7 45 24" src="https://github.com/woowacourse/javascript-lunch/assets/157036488/c7dd6f45-9528-4fd4-b2c2-d8c0118bb6b0">

as const를 해주면 그 값 자체로 추론한다. 
<img width="568" alt="스크린샷 2024-03-09 오후 7 45 46" src="https://github.com/woowacourse/javascript-lunch/assets/157036488/0caa07b5-d56c-4626-863b-552d84608201">

(+) (Object.freeze 해주는 것과 비슷한 효과! 그러나 차이점은 Object.freeze 와 달리 Const Assertions을 사용하면 깊은 동결을 할 수 있다!)
<img width="424" alt="스크린샷 2024-03-09 오후 8 05 02" src="https://github.com/woowacourse/javascript-lunch/assets/157036488/67ac95fd-fb17-4c8e-9549-7d2e1feb25f1">

평소에 상수는 Object.freeze()로 관리하는 편이었는데, 이번에 미션을 급하게 마무리하면서 이 부분을 놓친 것 같습니다. 그런데 덕분에 타입스크립트에서 더 편하고 엄격하게 상수를 관리할 수 있는 방법을 알게된 것 같아 좋네요! 꼼꼼하게 리뷰해주시고, 공부할 수 있도록 이끌어주셔서 감사합니다!!ㅎㅎ

Reply:
오 꼼꼼히 학습하셨군요 ㅎㅎ 멋집니다 헤인!